pgbouncer/tests/conftest.py

```diff
@@ -0,0 +1,87 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from __future__ import print_function
+import subprocess
+import os
+import time
+
+import pytest
+import psycopg2
+
+from . import common
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+def wait_for(service_name, port):
+    """
+    Try to connect to postgres/pgbouncer
+    """
+    for _ in range(10):
+        try:
+            psycopg2.connect(host=common.HOST, port=port, user=common.USER, password=common.PASS,
+                             database=common.DB, connect_timeout=2)
+            print("{} started".format(service_name))
+            return True
+        except Exception:
+            print("Waiting for {}...".format(service_name))
+            time.sleep(1)
+
+    return False
+
+
+@pytest.fixture(scope="session")
```

Comment:
In order to avoid having to make this a test function argument, let's make this `@pytest.fixture(scope="session", autouse=True)`.